index.html

```diff
@@ -1,14 +1,41 @@
 <!DOCTYPE html>
 <html>
+
 <head>
-    <meta charset='utf-8'>
-    <meta http-equiv='X-UA-Compatible' content='IE=edge'>
-    <title>Page Title</title>
-    <meta name='viewport' content='width=device-width, initial-scale=1'>
-    <link rel='stylesheet' type='text/css' media='screen' href='main.css'>
-    <script src='main.js'></script>
+  <meta charset="utf-8">
+  <title>Studio Lenzing</title>
+  <link rel="stylesheet" type="text/css" href="bootstrap.css">
```

Comment:
Use cdn and practice to rely on Bootstrap's docs https://getbootstrap.com/docs/4.3/getting-started/introduction/

Reply:
This is noted.

Reply:
Updated to use CDN instead of local css file.